assets/component-slider.css

```diff
@@ -132,30 +108,20 @@ slider-component .slider-buttons {
 
 .slider-button {
   color: rgba(var(--color-foreground), 0.75);
-  border: 0.1rem solid rgba(var(--color-foreground), 0.08);
-  background-color: rgb(var(--color-background));
+  background: transparent;
+  border: none;
   cursor: pointer;
   width: 44px;
   height: 44px;
 }
 
 .slider-button:not([disabled]):hover {
   color: rgb(var(--color-foreground));
```

Comment:
I wonder if we should be adding `transition` here like we are for other animations on hover.